bcauth/jinja2/oauth2_provider/application_confirm_delete.html

```diff
@@ -0,0 +1,18 @@
+{% extends "account/profile.html" %}
+
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+
+{% block content %}
+    <h3>{{ _("Are you sure to delete the application %(app_name)s?") | format(app_name=application.name) }}</h3>
+    <form method="post" action="{{ url('oauth2_provider:delete', application.pk) }}">
+        {{ csrf_input }}
+
+        <div class="control-group">
+            <div class="controls">
+                <a class="btn btn-large" href="{{ url("oauth2_provider:list")}}">{{ _("Cancel") }}</a>
```

Comment:
Will fix. I'm blind to it, but you and @jezdez are not. Any linter tools to auto-find spacing in jinja templates?